src/App.css

```diff
@@ -1,64 +1,74 @@
+.body,
+html {
+    overflow-x: hidden;
+}
+
 .App {
-  text-align: center;
+    text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.App-main {
+    margin: 0 auto;
+    padding: 50px 80px;
 }
 
-.profile{
-  display: flex;
-  align-items: flex-start;
-  justify-content: space-between;
-  margin: 50px 0px 0px;
-  padding-bottom: 50px;
+.profile {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    margin: 50px 20px 0px;
+    padding-bottom: 50px;
+    text-align: start;
+    font-size: 20px;
+    padding-bottom: 50px;
 }
 
-.profile-title{
-  width: 50%;
+.profile-title {
+    width: 50%;
 }
 
-.profile-title h1{
-  font-size: 40px;
-  font-weight: 700;
-  color: gray;
+.profile-title h1 {
+    font-size: 40px;
+    font-weight: 700;
+    color: gray;
 }
 
-.profile-description{
-  width: 50%;
-  display: flex;
-  flex-direction: column;
-  justify-content: start;
-  
+.profile-title p {
+    font-weight: 700;
+    font-size: 25px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.profile-description {
+    width: 50%;
 }
 
+.profile-description p {
+    width: 95%;
+    line-height: 1.9;
+    font-weight: 500;
+}
+
+.profile-description strong {
+    color: gray;
+    font-weight: 800;
+    font-size: 1.3em;
+}
 .App-header {
-  background-color: #282c34;
-  min-height: 100px;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+    background-color: #282c34;
+    min-height: 100px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
```

Comment:
use the same values for everything .. for example use `em` or `rem` for all font sizes .. other wise you might run into funny issues :D

Reply:
ok, this is was the default in react app, i'll change it.